tests/ImageSharp.Tests/Formats/Tiff/TiffEncoderTests.cs

```diff
@@ -473,6 +513,7 @@ public void TiffEncoder_EncodeBiColor_WithModifiedHuffmanCompression_BlackIsZero
 
     [Theory]
     [WithFile(GrayscaleUncompressed, PixelTypes.L8, TiffPhotometricInterpretation.BlackIsZero, TiffCompression.PackBits)]
+    [WithFile(GrayscaleUncompressed, PixelTypes.L16, TiffPhotometricInterpretation.BlackIsZero, TiffCompression.PackBits)]
```

Comment:
I need to generate L16 versions of `GrayscaleUncompressed` and `Calliphora_GrayscaleUncompressed`. What's the recommended way to do so?

Reply:
@brianpopow You generated the tiff files didn't you?

Reply:
I usually use  `tiffcp` which comes with `libtiff` to convert tiff, but I dont think you can change the bit per pixel in the output. It will have the same as the input (maybe I am missing an option).
```
usage: tiffcp [options] input... output
```

If `tiffcp` does not work, I use `imagemagick`. You can install `imagemagick` via [chocolatey](https://chocolatey.org)
The command would be:
```
magick convert Calliphora_grayscale_uncompressed.tiff -depth 16 Calliphora_grayscale_uncompressed_16bit.tiff
```

To convert with compression, use the `-compress` flag.

To make sure the image is as expected, I again use a tool from `libtiff` called `tiffinfo`. It shows a nice summary of the tiff.

Reply:
Thanks! I was able to use `magick` for depth, and `tiffcp` for compression.